pyseir/models/suppression_policies.py

```diff
@@ -22,6 +22,31 @@
 }
 
 
+def get_future_supression_from_r0(R0, scenario):
+    """
+    Returns the future suppression level for a given R0 and a "future scenario".
+
+    Parameters
+    ----------
+    R0:float
+        Reproduction number
+    scenario: str
+        'no_intervention', 'flatten_the_curve', 'social_distancing'.
```

Comment:
would be nice to add enum for this.

Reply:
agree. they are about to change so will do in a broader cleanup.